src/main/java/seedu/letsgethired/logic/commands/AddCommand.java

```diff
@@ -25,11 +26,13 @@ public class AddCommand extends Command {
             + PREFIX_COMPANY + "COMPANY "
             + PREFIX_ROLE + "ROLE "
             + PREFIX_CYCLE + "CYCLE "
+            + PREFIX_DEADLINE + "DEADLINE "
```

Comment:
could we include some way to tell the user the deadline format here ? apart from the example given below

Reply:
Actually, I put the constraint message over here upon user typing in the incorrect format! Would this be okay?
![image](https://github.com/AY2324S1-CS2103T-W17-2/tp/assets/107602488/279c5d10-e2cc-4edc-a235-6eb057828c2f)

Reply:
Also, I have updated the `formatDisplay` and `formatFeedback` method. Thanks for the catch!

Reply:
LGTM, deadline error message looks good